composer.json

```diff
@@ -60,9 +60,10 @@
   },
   "require-dev": {
     "phpunit/phpunit": "7.*",
+    "phpunit/php-invoker": "^2.0",
```

Comment:
Why do we Need this here?

Reply:
phpunit advised to install this (error: php-invoker blabla)

Reply:
Package phpunit/php-invoker is required for enforcing time limits